### PR TITLE
Annual Reviews are going away as of September 2023

### PR DESCRIPTION
**Pull request by amye**
Deprecating annual reviews in favor of an automated process!

### CODE REVIEW

**Comment by amye** [comment]
> I'm happy to see this! However, what is the "proposed automated review system"? :) As a non-TOC member this is very vague

We're working with the LFX team to define this, it'll be based off of https://github.com/cncf/toc/issues/1134

**Comment by tomkerkhove** [comment]
Thanks for sharing. Can we add a pointer to that in the doc?

**Comment by amye** [comment]
The hope is that the issue will be completed by the time we merge these in.

File: process/README.md [hunk 1]
```diff
@@ -110,20 +110,4 @@ What does archiving for a CNCF project mean?
...
-* What are the current goals of the project? For example, are you working on major new features? Or are you concentrating on adoption or documentation?
-* How can the CNCF help you achieve your upcoming goals?
-* Do you think that your project meets the criteria for incubation?
-
-Based on this review the TOC will vote on whether to continue to sponsor the project in Sandbox. If the vote does not reach a simple majority, we will discuss archiving or other next steps with you.
-
-Additionally, the TOC might recommend that you apply for Incubation stage. This requires extra work and due diligence so it’s not a possible outcome to move directly to Incubation from this lightweight annual review.
-
-It is fine for a project to stay in the Sandbox indefinitely while it is still active, but if a project has genuinely stalled we can save everyone’s effort by archiving it.
+The TOC is deprecating the current annual review process as of September 2023 in favor of a proposed automated review system.
```
Comment (AkihiroSuda) [inline process/README.md:113]: Could you add a link to the "proposed automated review system" ?

Comment (amye) [reply process/README.md:113]: As soon as we have that, I will share!

**Comment by makkes** [comment]
> The hope is that the issue will be completed by the time we merge these in.

It's been nearly 4 months now since this PR has been merged but https://github.com/cncf/toc/issues/1134 still isn't merged. I assume that as a consequence projects will not be reviewed at all. It'd be great to hear from TOC on this.

**Pull request merged**